src/styles/stars.scss

```diff
@@ -0,0 +1,34 @@
+.review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 16px;
+
+  &__stars {
+    display: flex;
+    align-items: center;
+
+    &--star {
+      background-image: url('../images/star.svg');
+      background-repeat: no-repeat;
+      background-position: center;
+      height: 16px;
+      width: 16px;
+      margin-right: 4px;
+
+      &:nth-child(-n + 4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+
+  &__text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-text-color;
```

Comment:
The variable `$main-text-color` is used here. Ensure that it is defined in your SCSS files to prevent any compilation errors.